docs/devel/generating-clientset.md

```diff
@@ -48,7 +48,8 @@ the types (e.g., Pods) that you want to generate clients for with the
 namespace scoped (e.g., PersistentVolume), you need to append the
 `nonNamespaced=true` tag as well.
 
-- Running the client-gen tool: you need to use the command line argument
+- Running the client-gen tool: this is done [in a script](../../hack/update-codegen.sh).
```

Comment:
It suggested the modification because people may use client-gen as a stand-alone tool.